content/applications/general/users.rst

```diff
@@ -1,161 +1,20 @@
-:show-content:
```

Comment:
No, it's on purpose, I don't want to show the content.
We could remove the content too, but it does not hurt if you go to this page from another link.

Reply:
That's really bad behavior. The user can access the page from the "User docs" table of content but can't go back to it after navigating further. We'll leave it that way for now but I'm sure that this page will undergo additional changes soon.

Reply:
okay I see now it's done on purpose. I delete my previous comment.
This doesn't look good when we end up on "users" or "auth" from the "applications.html" page.
I'm even considering doing a temporary manual list instead of using `:show-toc:` on these pages.